VKSample/VKSample/Modules/FriendsModule/FriendsTableViewController.swift

```diff
@@ -46,21 +49,44 @@ final class FriendsTableViewController: UITableViewController {
     // MARK: - Private methods.
 
     private func fetchFriends() {
-        networkService.fetchFriends { [weak self] result in
+        guard let objects = realmService.readData(items: Friend.self) else { return }
+        if !objects.isEmpty {
+            friends = objects
+            setupCellsToSections()
+        } else {
+            networkService.fetchFriends { [weak self] result in
+                guard
+                    let self = self,
+                    let friends = self.friends
+                else { return }
+                switch result {
+                case .success:
+                    self.createFriendsNotificationToken(resultFriends: friends)
+                case let .failure(error):
+                    self.showErrorAlert(title: Constants.errorTitleString, message: "\(error.localizedDescription)")
+                }
```

Comment:
```suggestion
 fetchFriends()
```
и все что удалили туда, а эта функция больше похожа на общую загрузку, loadFriends()